packages/node/src/config/ConfigWizard.ts

```diff
@@ -21,7 +21,7 @@ import * as BrokerConfigSchema from './config.schema.json'
 
 const MIN_BALANCE = parseEther('0.1')
 
-type EnvironmentId = 'polygon' | 'polygonAmoy'
+type EnvironmentId = 'polygon' | 'polygonAmoy' | 'peaq'
```

Comment:
Which environments the `ConfigWizard` should support? If we want peaq, it should be added to the `getEnvironmentId()` function at line 249.

Maybe this `EnvironmentId` could use sdk's `EnvironmentId` as base type?